packages/dso-toolkit/components/Voorbeelden/Homepage.njk

```diff
@@ -1,83 +1,211 @@
+                    <span class="dso-tile-icon">
+                      <img src="https://dso-toolkit.nl/37.0.0/graphics/erfafscheiding_plaatsen.svg" alt=""/>
+                    </span>
+                    <span class="dso-tile-link">
+                  Boom kappen of snoeien
+                </span>
+                  </a>
+             
+                </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+    </div>
     <div class="col-md-4 col-sm-6 col-xs-12">
-      {% render '@whitebox-regular', {title: vergunning.title, description: vergunning.description, linktext: vergunning.linktext, source: vergunning.source} -%}
+      <div class="dso-highlight-box dso-white dso-drop-shadow">
+        <div class="dso-rich-content">
+          <img src="../../dummy/images/hp7.png" class="img-responsive" aria-hidden="true">
+          <h2>Vergunning aanvragen of melding doen</h2>
+          <p>Weet u al voor welke activiteiten u een vergunning moet aanvragen of een melding moet doen? Dan kan u een aanvraag of melding starten.</p>
+          <a href="#"  class="btn dso-secondary">
+            <span>Aanvragen</span>
+            <svg class="di di-chevron-right">
+              <use href="/dso-icons.svg#chevron-right"/>
+            </svg>
+          </a>
+        </div>
+      </div>
     </div>
+
     <div class="col-md-4 col-sm-6 col-xs-12">
-      {% render '@whitebox-regular', {title: aanvragen.title, description: aanvragen.description, linktext: aanvragen.linktext , source: aanvragen.source} -%}
+      <div class="dso-highlight-box dso-white dso-drop-shadow">
+        <div class="dso-rich-content">
+          <img src="../../dummy/images/hp5.png" class="img-responsive"  aria-hidden="true">
+          <h2>Bekijk de regels op de kaart op uw locatie</h2>
+          <p>In Regels op de kaart vindt u de regels van alle overheden bij elkaar. U kunt ook plannen van de overheid bekijken voor de toekomst </p>
+          <a href="#"  class="btn dso-secondary">
+            <span>Regels op de kaart</span>
+            <svg class="di di-chevron-right">
+              <use href="/dso-icons.svg#chevron-right"/>
+            </svg>
+          </a>
+        </div>
+      </div>
     </div>
+
     <div class="col-md-4 col-sm-6 col-xs-12">
-      {% render '@whitebox-regular', {title: kaart.title, description: kaart.description, linktext: kaart.linktext, source: kaart.source} -%}
+      <div class="dso-highlight-box dso-white dso-drop-shadow">
+        <div class="dso-rich-content">
+          <img src="../../dummy/images/hp6.png" class="img-responsive"  aria-hidden="true">
+          <h2>Maatregelen voor bedrijven en burgers</h2>
+          <p>Welke algemene regels gelden er voor uw bedrijf, huis of project? Krijg een overzicht van de maatregelen waarmee u aan de regels kunt voldoen.</p>
+          <a href="#"  class="btn dso-secondary">
+            <span>Maatregelen op maat</span>
+            <svg class="di di-chevron-right">
```

Comment:
De ene keer maak je van `<dso-icon>` gebruik en de andere keer van `<svg class="di">`. Daarnaast werken de iconen niet omdat je naar `/dso-icons.svg` verwijst, maar die bestaat niet op dso-toolkit.nl

Reply:
Instanties van dso-icon verwijderd. Het is raar maar bij mij rendert het icoon die niet goed uit met de tekst als ik dso-icon in bepaalde situaties gebruik